earthsim/grabcut.py

```diff
@@ -201,22 +201,22 @@ def bg_path_view(self):
             self._bg_data = []
         elif self._initialized:
             self._bg_data = self.draw_bg.element.data
-        return gv.project(self.path_type(self._bg_data, crs=self.crs), projection=self.image.crs)
+        return gv.project(self.path_type(self._bg_data, crs=ccrs.GOOGLE_MERCATOR), projection=self.image.crs)
```

Comment:
@philippjfr This is really where we encountered the problem with the change in Quest to return images as GM. It looks like ``self._bg_data`` ends up being in the same projection as the image, so we needed to switch the declared ``crs`` here. I think we need to find a way to make this work independently of the ``image.crs``.

Reply:
I just pushed another commit that simplifies/corrects how the crs information is handled. I've also released Quest 3.1, so the CI builds should now be able to test this.